src/common/components/NFTCard/NFTCard.tsx

```diff
@@ -1,12 +1,11 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
 import cn from 'classnames'
+import Rarenessbar from '../Rarenessbar'
+import OnSaleImage from '../../assets/images/on_sale.png'
 
 export interface INFTCompactCardProps {
```

Comment:
Please use the Type instead of the Interface. @sanjanadhikari1

Reply:
fixed